pt-BR/mail/chrome/messenger/addressbook/abSelectAddressesDialog.dtd

```diff
@@ -36,32 +36,31 @@
  ***** END LICENSE BLOCK ***** -->
 
 <!-- Title -->
-<!ENTITY selectAddressWindow.title           "Select Addresses">
+<!ENTITY selectAddressWindow.title           "Selecione os endereços">
 
 <!-- Buttons -->
-<!ENTITY toButton.label                      "To->">
+<!ENTITY toButton.label                      "Para->">
 <!ENTITY toButton.accesskey                  "T">
 <!ENTITY ccButton.label                      "Cc->">
 <!ENTITY ccButton.accesskey                  "C">
-<!ENTITY bccButton.label                     "Bcc->">
+<!ENTITY bccButton.label                     "Cco->">
 <!ENTITY bccButton.accesskey                 "B">
-<!ENTITY newButton.label                     "New...">
+<!ENTITY newButton.label                     "Novo...">
 <!ENTITY newButton.accesskey                 "N">
-<!ENTITY editButton.label                    "Edit...">
+<!ENTITY editButton.label                    "Editar...">
 <!ENTITY editButton.accesskey                "E">
-<!ENTITY removeButton.label                  "Remove">
+<!ENTITY removeButton.label                  "Remover">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY removeButton.label                  "Excluir">